server/app/views/ApplicationBaseView.java

```diff
@@ -136,8 +139,9 @@ public abstract Builder setErrorDisplayMode(
    * @param messages the localized {@link Messages} for the current applicant
    * @return PTag containing requiredness text.
    */
-  public PTag requiredFieldsExplanationContent(Messages messages) {
-    return p(messages.at(MessageKey.REQUIRED_FIELDS_ANNOTATION.getKeyName()))
+  public static PTag requiredFieldsExplanationContent(Messages messages) {
+    SpanTag redAsterisk = span("*").withClass("text-red-700");
```

Comment:
**thought:** If this is a color we're going to start using more consistently, we can add it to the base styles with a constant name https://github.com/civiform/civiform/blob/f1f5a42151b63aee20210ddfda4a3778dd24c82e/server/app/views/style/BaseStyles.java#L45

Reply:
That's a good idea.  I'll add that style.

Reply:
Realized that the other red asterisks on the page were "text-red-600", so, for consistency, I'm going to use the already existing BaseStyle.FORM_ERROR_TEXT_COLOR that you mentioned.